docs/basic_training/cache_and_resume.md

```diff
@@ -200,6 +200,10 @@ Finally, the `-t` option enables the creation of a basic custom provenance repor
 
 ## Resume troubleshooting
 
+!!! tip
```

Comment:
I would drop this below the intro paragraph, and reword / rewrite the intro paragraph a little.

Also, may as well use a numbered list - easier to read than having them inline text.

Reply:
I agree with Phil.